boards/nrf51dk/src/main.rs

```diff
@@ -261,15 +261,35 @@ pub unsafe fn reset_handler() {
 
 
 use core::fmt::Arguments;
+use core::fmt::Write;
+use core::fmt::write;
 #[cfg(not(test))]
 #[lang="panic_fmt"]
 #[no_mangle]
-pub unsafe extern "C" fn rust_begin_unwind(_args: &Arguments,
+pub unsafe extern "C" fn rust_begin_unwind(_args: Arguments,
```

Comment:
Also, every other board names it as `panic_fmt`. This is slightly relevant because the compiler flag `#[lang="panic_fmt"]` actually changes the name to be `panic_fmt` regardless of what we call it, meaning confusion for anyone searching through a disassembly for this function gets very confused.